network/endpoint_linux.go

```diff
@@ -304,6 +302,23 @@ func addRoutes(nl netlink.NetlinkInterface, netioshim netio.NetIOInterface, inte
 			Table:     route.Table,
 		}
 
+		if removeRouteBeforeAdd {
```

Comment:
Is this a part of adding constant mac ? 
 This is used by transport vlan mode only, shouldn't we delete first as separate call and then add routes. Since the function now is adding and deleting routes. (separation of concerns)

Reply:
yep agreed. let me fix it